src/thorlabs_cube/driver/tcube/tsc.py

```diff
@@ -0,0 +1,407 @@
+import struct as st
+
+from thorlabs_cube.driver.base import _Cube
+from thorlabs_cube.driver.message import MGMSG, Message, MsgError
+
+_CHANNEL = 0x01
+_RESERVED = 0x00
+
+
+class Tsc(_Cube):
+    """TSC001 T-Cube Motor Controller class"""
+
+    def __init__(self, *args, **kwargs):
+        _Cube.__init__(self, *args, **kwargs)
+        self.status_report_counter = 0
+
+    async def handle_message(self, msg):
+        msg_id = msg.id
+        data = msg.data
+
+        if msg_id == MGMSG.HW_DISCONNECT:
+            raise MsgError("Error: Please disconnect the TDC001")
+        elif msg_id == MGMSG.HW_RESPONSE:
+            raise MsgError(
+                "Hardware error, please disconnect " "and reconnect the TDC001"
```

Comment:
```suggestion
                "Hardware error, please disconnect and reconnect the TDC001"
```
Awkward string representation.